### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by kylecalbert**
Created a Card component with repsonsive Grid. Refactored some of the px values to rem/em for better responsiveness.
Screenshot 2023-09-21 at 09 55 23

### CODE REVIEW

**Comment by wingleungchoi** [comment]
LGTM
Good job! 👍

**Review by wingleungchoi — APPROVED** [review]

**Comment by wingleungchoi** [comment]
In order to learn how npm and js library work, you could consider to build and publish js library of it.
the project looks nothing related to sensitive data, so it should be safe to publish in npm.

File: src/components/Content/Card.js [hunk 1]
```diff
@@ -1,27 +1,83 @@
...
-import Icon from '../Icon/Icon'; // Import the corrected Icon component
 import { darkModeColors } from '../../constants/colors';
 import { Footer } from '../Footer/Footer';
+
+import CalendarDark from '../../Assets/dark/calendar.png';
+import LocationDark from '../../Assets/dark/location.png';
+import { Image } from '../Image/Image';
+
+//create constant for resued sizes
+const SemiBold = '600';
```
Comment (sukhvir-notta) [inline src/components/Content/Card.js]: Could create a separate constants file if these sizes are also being used in other components

**Review by sukhvir-notta — APPROVED** [review]

File: src/components/Content/Card.js (hunk 1 above)
Comment (kylecalbert) [reply src/components/Content/Card.js]: Done :)

**Pull request merged**